pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```diff
@@ -333,6 +333,10 @@ runCommandLocal name
             '';
         inherit args fhsenv;
       };
+
+      meta = {
+        mainProgram = executableName;
+      } // meta;
```

Comment:
This means that the underlying meta.mainProgram will override this mainProgram. Is that what you meant?

Reply:
Seems fine to me.

My question comes about because there's a little dance that these sort of "defaults" often play. They want to make sure that the executable name _for sure_ is a value that's going to work, and so they want the opposite semantics.

```
meta = meta // { mainProgram = executableName; }
```

The use case here, as I understand it, is to make `lib.getExe` work, along with other functions that rely on `meta.mainProgram` being set. Is that right?

Reply:
Yes, that's the use case.

I wrote it like this because I thought people should be able to override something if they really want to.